saferpayofficial.php

```diff
@@ -347,8 +347,36 @@ public function hookActionFrontControllerSetMedia()
         /** @var \Invertus\SaferPay\Presentation\Loader\PaymentFormAssetLoader $paymentFormAssetsLoader */
         $paymentFormAssetsLoader = $this->getService(\Invertus\SaferPay\Presentation\Loader\PaymentFormAssetLoader::class);
 
+        /** @var \Invertus\SaferPay\Action\ValidateOpcModuleCompatibilityAction $validateOpcModuleCompatibility */
+        $validateOpcModuleCompatibility = $this->getService(\Invertus\SaferPay\Action\ValidateOpcModuleCompatibilityAction::class);
+
         $paymentFormAssetsLoader->register($this->context->controller);
 
+        $opcModules = $validateOpcModuleCompatibility->run();
+
+        if ($opcModules) {
```

Comment:
why if needed ? just do the switch and add assets dependant of the result of opc modules